singer-io/tap-twitter-ads
# Tdl 18465 invalid bookmark for tweets stream

# Description of change
- Added logic to separate out bookmarks for each tweet type.
- Updated bookmark logic for `SCHEDULED` tweets type.
- Added integration test case.

# Manual QA steps
 - Run sync mode and verify that tap writes a separate bookmark for each tweets_type.
 - Run sync mode with state verify that each tweet_type respect their bookmark.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
